testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_replication.py

```diff
@@ -2981,8 +2981,11 @@ def test_resetCheckpointFailure(params_from_base_test_setup):
     # verify it throws an error that checkpoint reset is called without stopping replicator.
     with pytest.raises(Exception) as he:
         replicator.resetCheckPoint(repl)
-    assert 'Replicator is not stopped.' in he.value.message
-    assert 'Resetting checkpoint is only allowed when the replicator is in the stopped state' in he.value.message
+    if liteserv_platform.lower() == "android":
+        assert 'Attempt to reset the checkpoint for a replicator that is not stopped.' in he.value.message
+    else:
+        assert 'Replicator is not stopped.' in he.value.message
```

Comment:
can we add the explanatory statement to tell what caused the failure in asserts?